enclave/core/sgx/writebarrier.c

```diff
@@ -5,6 +5,8 @@
 #include <openenclave/enclave.h>
 #include <openenclave/internal/raise.h>
 
+#include <openenclave/internal/print.h>
```

Comment:
This include does not seem related to this PR.

Reply:
Fixed. This was for verifying the fix.